### ISSUE TITLE
Error in get_timestamps_and_queries in v0.0.2

### ISSUE THREAD

**Issue by chrismessina** [mr-pennyworth/alfred-history-search#2]
Installed the newest version, his the shortcut, and got this error (and no result in Alfred):

```
[22:30:57.340] Alfred History Search[Hotkey] Processing complete
[22:30:57.348] Alfred History Search[Hotkey] Passing output '' to Script Filter
[22:30:57.374] Alfred History Search[Script Filter] Queuing argument '(null)'
[22:30:57.546] Alfred History Search[Script Filter] Script with argv '(null)' finished
[22:30:57.554] ERROR: Alfred History Search[Script Filter] Code 1: Traceback (most recent call last):
  File "search_history.py", line 76, in <module>
    ts_n_qs = get_timestamps_and_queries()
  File "search_history.py", line 49, in get_timestamps_and_queries
    timestamp, query = re.findall(regex, line)[0]
IndexError: list index out of range
```

**Comment by mr-pennyworth** [comment]
Dang!
Thanks for reporting @chrismessina!
Looks like it is erroring whenever there's been a multi-line query in Alfred.

**Comment by mr-pennyworth** [comment]
Fixed in `0.0.3`
